politeiawww/prices.go

```diff
@@ -12,33 +13,69 @@ import (
 	database "github.com/decred/politeia/politeiawww/cmsdatabase"
 )
 
+const binanceURL = "https://api.binance.com"
 const poloURL = "https://poloniex.com/public"
 const httpTimeout = time.Second * 3
 const pricePeriod = 900
 
+const dcrSymbolBinance = "DCRBTC"
+const usdtSymbolBinance = "BTCUSDT"
+
+const dcrSymbolPolo = "BTC_DCR"
+const usdtSymbolPolo = "USDT_BTC"
+
 type poloChartData struct {
 	Date            uint64  `json:"date"`
 	WeightedAverage float64 `json:"weightedAverage"`
 }
 
+// Set the last date to use polo as 4/1/2019.  If any start date requested is
+// after that date then use Binance instead.
+var endPoloDate = time.Date(2019, 4, 1, 0, 0, 0, 0, time.UTC)
+
 // GetMonthAverage returns the average USD/DCR price for a given month
 func (p *politeiawww) GetMonthAverage(month time.Month, year int) (uint, error) {
```

Comment:
Not part of this PR, but I'm just noticing it now.  This function is in package main so there's not reason to export it.